omnibus/files/private-chef-ctl-commands/cleanup_bifrost.rb

```diff
@@ -0,0 +1,213 @@
+require 'pg'
+
+CREATE_SQL =<<SQL
+BEGIN;
+CREATE TABLE IF NOT EXISTS cleanup_tracking_auth_actors(
```

Comment:
Interesting!  I definitely think that we should used `UNLOGGED`.  I'll need to look into how the PG gem manages the connection to know if TEMPORARY will actually work for us.